src/ogt_vox.h

```diff
@@ -1860,13 +1886,22 @@
             ogt_vox_instance new_instance;
             new_instance.model_index                = 0;
             new_instance.group_index                = 0;
-            new_instance.transform                  = _vox_transform_identity();
+            new_instance.transform                  = ogt_vox_transform_get_identity();
             new_instance.layer_index                = 0;
             new_instance.name                       = 0;
             new_instance.hidden                     = false;
             clear_anim_transform(&new_instance.transform_anim);
             clear_anim_model(&new_instance.model_anim);
             instances.push_back(new_instance);
+            // adds a single group
+            ogt_vox_group new_group;
+            new_group.hidden = false;
+            new_group.layer_index = 0;
+            new_group.transform = ogt_vox_transform_get_identity();
+            new_group.parent_group_index = k_invalid_group_index;
+            clear_anim_transform(&new_group.transform_anim);
+            new_group.name = 0;
```

Comment:
shouldn't this be `NULL`?

Reply:
It's a little subtle but at this point, name is just an offset into the misc_data blob that contains all the strings, where an offset of zero means unallocates. The pointer for name is patched up later.